### PR TITLE
added tags for some problems

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/2_Bronze/Ad_Hoc.problems.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
       "source": "Bronze",
       "difficulty": "Hard",
       "isStarred": false,
-      "tags": [],
+      "tags": ["Implementation"],
```
Comment (SansPapyrus683) [inline content/2_Bronze/Ad_Hoc.problems.json:38]: idt this is a tag

Comment (freakin23) [reply content/2_Bronze/Ad_Hoc.problems.json:38]: ig Brute force will work for this one.

Comment (SansPapyrus683) [reply content/2_Bronze/Ad_Hoc.problems.json]: that's not a tag either

Comment (freakin23) [reply content/2_Bronze/Ad_Hoc.problems.json]: its "Complete Search"

Comment (freakin23) [reply content/2_Bronze/Ad_Hoc.problems.json]: I have updated it to "Complete Search". Hope it works

**Review by SansPapyrus683 — APPROVED** [review]
i swear to god if someone opens an issue saying "this tag doesn't fit" i'm committing a felony

**Comment by freakin23** [comment]
> i swear to god if someone opens an issue saying "this tag doesn't fit" i'm committing a felony

sarcasm? 
At first I was going for "Ad-hoc" but I don't consider  much of a type. Since, the solution is pretty much brute force that's why I guess complete search is appropriate for it.

**Pull request merged**